docs/primer.md

```diff
@@ -262,6 +262,18 @@ In addition an element has the optional properties `notNull`, and `default` (bot
 
 Note: As a general convention we recommend to omit optional boolean properties, if the value is `false`.
 
+### Cardinality
+
+Please note that CSN Interop does not support arrays of scalar types, only of [cds.Association](./spec-v1/csn-interop-effective#association-type) and [cds.Composition](./spec-v1/csn-interop-effective#composition-type), which can express this in their [cardinality](./spec-v1/csn-interop-effective#cardinality-object) object.
```

Comment:
this sounds like CSN interop supports "arrays of cds.Association/Composition", which is not the case

Reply:
yes, what I meant is that cdsAssociation has cardinality ("to many"), which is essentially an array of pointers. 

Any suggestions how to phrase it better?

```suggestion
Please note that CSN Interop does not support arrays of scalar types. There is [cds.Association](./spec-v1/csn-interop-effective#association-type) and [cds.Composition](./spec-v1/csn-interop-effective#composition-type), where (to many) [cardinality](./spec-v1/csn-interop-effective#cardinality-object) can be defined.
```

Reply:
@stewsk - I've updated the descriptions, is it better now?